File: DropZone/DropZone.Core/Models/ParachuteSystem/ParachuteSystemModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DropZone.Core.Models
{
    public class ParachuteSystemModel
    {
        public long Id { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long MainParachuteId { get; set; }

        public long ReserveParachuteId { get; set; }

        public long AADId { get; set; }

        public long SatchelId { get; set; }

        public long UserId { get; set; }

        public long HashBlockId { get; set; }


        public ParachuteModel MainParachute { get; set; }

        public ParachuteModel ReserveParachute { get; set; }

        public AADModel AAD { get; set; }

        public SatchelModel Satchel { get; set; }

        public UserModel User { get; set; }

        public HashBlockModel HashBlock { get; set; }
    }
}

File: DropZone/DropZone.Web/src/models/parachuteSystems/parachuteSystem.model.ts
import { ParachuteModel } from "../parachutes/parachute.model";
import { AADModel } from "../aads/aad.model";
import { SatchelModel } from "../satchels/satchel.model";
import { UserModel } from "../users/user.model";
import { HashBlockModel } from "../hashBlocks/hashBlock.model";

export class ParachuteSystemModel {
	public Id: number;
    public ModifiedAt: Date;

	public MainParachuteId: number;
	public ReserveParachuteId: number;
	public AADId: number;
	public SatchelId: number;
	public UserId: number;
    public HashBlockId: number;
    
	public MainParachute: ParachuteModel;
	public ReserveParachute: ParachuteModel;
	public AAD: AADModel;
	public Satchel: SatchelModel;
	public User: UserModel;
    public HashBlock: HashBlockModel;
};